memcached/memcached.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kubedb.com/v1
+kind: Memcached
```

Comment:
Is tls supported ? If yes, add that too in a seperate folder. And a small read me about creating CA

Reply:
Currently, TLS is not supported.